.github/workflows/merge.yml

```diff
@@ -593,6 +593,297 @@ jobs:
           log_file_prefix: ant_test_logs_e2e
           build: true
 
+  pointer:
+    name: Pointer tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            ant_path: /home/runner/.local/share/autonomi
+          - os: windows-latest
+            ant_path: C:\\Users\\runneradmin\\AppData\\Roaming\\autonomi
+          - os: macos-latest
+            ant_path: /Users/runner/Library/Application\ Support/autonomi
+    steps:
+      - uses: actions/checkout@v5
+
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+      - uses: Swatinem/rust-cache@v2
+
+      - name: Build binaries
+        run: cargo build --release --bin antnode --bin ant --bin antctl
+        timeout-minutes: 30
+
+      - name: Install antctl on Windows
+        if: matrix.os == 'windows-latest'
+        shell: pwsh
+        run: |
+          $destination = "C:\\Users\\runneradmin\\AppData\\Local\\Microsoft\\WindowsApps"
+          Copy-Item "target\\release\\antctl.exe" -Destination $destination -Force
+          Write-Host "antctl installed to $destination"
+
+      - name: Start a local network
+        uses: maidsafe/ant-local-testnet-action@main
+        with:
+          action: start
+          enable-evm-testnet: true
+          node-path: target/release/antnode
+          build: true
+
+      - name: Check if ANT_PEERS and EVM_NETWORK are set
+        shell: bash
+        run: |
+          if [[ -z "$ANT_PEERS" ]]; then
+            echo "The ANT_PEERS variable has not been set"
+            exit 1
+          elif [[ -z "$EVM_NETWORK" ]]; then
+            echo "The EVM_NETWORK variable has not been set"
+            exit 1
+          else
+            echo "ANT_PEERS has been set to $ANT_PEERS"
+            echo "EVM_NETWORK has been set to $EVM_NETWORK"
+          fi
+
+      # FIXME: do this in a generic way for localtestnets
+      - name: export default secret key
+        if: matrix.os != 'windows-latest'
+        run: echo "SECRET_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80" >> $GITHUB_ENV
+        shell: bash
+      - name: Set secret key for Windows
+        if: matrix.os == 'windows-latest'
+        run: echo "SECRET_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+        shell: pwsh
+
+      # Create test data files for pointer targets
+      - name: Create test data files
+        run: |
+          echo "test_data_A" > test_data_A.txt
+          echo "test_data_B" > test_data_B.txt
+
+      # Upload test data to get addresses for pointer targets
+      - name: Upload test data A
+        run: ./target/release/ant --log-output-dest=data-dir --local file upload --public test_data_A.txt > ./upload_output_A 2>&1
```

Comment:
As per scratchpads, I don't think we need the `--log-output-dest` on the commands.